profile/README.md

```diff
@@ -1,36 +1,60 @@
+A browser-based interface to explore, select, and analyze scenes using FTW models and data. 
 
-3. **Multi-class Segmentation**: The dataset provides masks for both instance segmentation and semantic segmentation with different classes, including:
-   - **Instance Segmentation Masks**: To identify individual fields.
-   - **Semantic Segmentation Masks**: 
-     - Two-class masks: Background and polygon (field).
-     - Three-class masks: Background, polygon (field), and boundaries.
+### 📡 FTW Source Data  
+Curated public datasets of manually labeled field boundaries—such as those hosted on [fiboa.org](https://fiboa.org)—that may serve as inputs to the benchmark dataset. These are raw sources prior to FTW formatting.
 
-4. **Spectral Richness**: The dataset includes RGB (Red, Green, Blue) and NIR (Near-Infrared) spectral bands from Sentinel-2 images.
+### 🗺️ FTW Data Products  
+Large-scale outputs of model-inferred field boundaries (e.g., country-wide field maps). These are versioned and published for external research, planning, and applications.
 
-5. **Temporal Richness**: The dataset includes multi-date imagery to capture different stages of the growing season. Two images with distinct contrast differences were selected to represent these stages. To determine the date ranges for these images, the USDA Crop Calendar was initially referenced and then refined by selecting periods with minimal cloud cover and optimal contrast between the two images.
+---
 
-6. **Comprehensive Data Splits**: The dataset is carefully divided into training, validation, and test sets to ensure accurate evaluation of model performance. For each country, larger tiles are divided into smaller chips measuring 1536x1536 m². To prevent data leakage due to spatial autocorrelation, a blocked random splitting strategy is used. Chips are grouped into 3×3 blocks, with 80% allocated to training, 10% to validation, and 10% to testing.
+## 🤝 Get Involved
 
-7. **Metadata and Documentation**: The metadata and documentation provide crucial information to help users effectively interpret and utilize the dataset. It includes key details about the country of focus, temporal data collection windows, grid structures, and the year of collection.
+We welcome contributions of:
+- Field boundary datasets
+- Model improvements
+- Tool development
+- Promotion and educational materials (blog posts, documentation, talks, papers, social media)
+- Feedback and ideas
 
-   1. **Country**: The geographic region the dataset focuses on.
-   2. **Crop Types**: Crop types used to filter the polygons in the dataset, with specific keywords used for filtering.
-   3. **Seasons**: Date ranges defining the temporal windows for data collection.
-   4. **Year of Collection**: The year when the polygon boundaries were captured.
-   5. **Grids**: Larger grids from which smaller chips are derived, with some grids spatially separated to optimize area coverage.
+🧑‍💻 Check out our repositories  
```

Comment:
nit should this have a link to different repos?

Reply:
@m-mohr Which repos would you point people to for each of these contribution options?

Reply:
I guess you've listed it already in "What's in the FTW Ecosystem?", but without actual links to the repos?! :-)

Reply:
Yeah, I think the links should be in the FTW ecosystem.